packages/f-checkout/test/specs/component/f-checkout.component.spec.js

```diff
@@ -34,4 +34,20 @@ describe('f-checkout component tests', () => {
         CheckoutComponent.selectFulfillmentTime('As soon as possible');
         CheckoutComponent.submit();
     });
+
+    it('should display times in ascending order, with default text "As soon as possible" showing first', () => {
+        // Assert
+        expect(CheckoutComponent.isDropdownDisplayed()).toBe(true);
+
+        // Act
+        CheckoutComponent.selectOption().forEach((element) => {
+            element.click();
```

Comment:
What does clicking on each element test? That it is clickable? If so, that might be worth pulling out into a test of its own